### PR TITLE
Clear chat logs / history (#719 - 3)

### PR DESCRIPTION
**Pull request by draziw-**

### CODE REVIEW

**Comment by tsudoko** [comment]
Shouldn't removing history require confirmation?

**Comment by draziw-** [comment]
Actually, removing a client do not require confirmation (not only on uTox).

(How do you see such a confirmation system, a popup ?)

**Comment by tsudoko** [comment]
I think a popup is fine. And yes, I'm aware that removing other contacts doesn't require confirmation, that's a reported issue - #551. I'm not sure if there's a good crossplatform way of doing it.

File: list.c [hunk 1]
```diff
@@ -529,11 +529,16 @@ _Bool list_mdown(void *UNUSED(n))
 static void contextmenu_list_onselect(uint8_t i)
 {
     if(ritem->item == ITEM_FRIEND_ADD && i == 0) {
-        FRIENDREQ *req = ritem->data;
-        tox_postmessage(TOX_ACCEPTFRIEND, 0, 0, req);
-        return;
+		FRIENDREQ *req = ritem->data;
```
Comment (GrayHatter) [inline list.c]: Lines 532-534 are tabs instead of spaces.

**Comment by GrayHatter** [comment]
Every line changed is using tabs instead of the current style of 4 spaces for each indentation.

**Comment by GrayHatter** [comment]
#719

**Pull request merged**